### ISSUE TITLE
Setting the coordinate system in xESMF

### ISSUE THREAD

**Issue by ChristianSteger** [JiaweiZhuang/xESMF#39]
Hi Jiawei,

First of all, thank a lot for developing and maintaining this great package. I was was wondering if it is possible to switch between planar and spherical coordinates in xESMF. I guess it is important how the coordinates are interpreted (as planar or spherical) when e.g. area-weights are computed for the conservative regridding. It seems possible to set this in ESMPy (see: http://www.earthsystemmodeling.org/esmf_releases/last_built/esmpy_doc/html/CoordSys.html#ESMF.api.constants.CoordSys.SPH_DEG). Is this feature also available in xESMF (and if so, how?)?

Cheers & thanks,
Christian

**Comment by JiaweiZhuang** [comment]
You are right that ESMF can deal with both spherical and cartesian coordinates. But I only specify spherical coordinate in [xESMF code](https://github.com/JiaweiZhuang/xESMF/blob/master/xesmf/backend.py#L102). I rarely use the cartesian one but I guess it should also work. You can try changing the code to `coord_sys=ESMF.CoordSys.CART`, and xESMF should interpret the 'lon' coordinate as 'x' in cartesian plane.

I am curious what's the use case of this? Are you dealing with non-geospatial data?
